public/app/features/dashboard/components/DashboardSettings/GeneralSettings.tsx

```diff
@@ -127,3 +134,14 @@ export const GeneralSettings: React.FC<Props> = ({ dashboard }) => {
     </div>
   );
 };
+
+const mapDispatchToProps = {
+  updateTimeZone: (timeZone: TimeZone) => {
+    updateTimeZoneForSession(timeZone);
+    getTimeSrv().refreshDashboard();
+  },
+};
+
+const connector = connect(null, mapDispatchToProps);
+
+export const GeneralSettings = connector(GeneralSettingsUnconnected);
```

Comment:
Don't think we need to use connector & mapDispatchToProps, we can just use the hook useDispatch directly from the component function

Reply:
This makes it easier to test I guess?

Reply:
Yes, exactly, I followed this pattern to make testing easier.